src/components/HexInput.tsx

```diff
@@ -50,4 +50,4 @@ const HexInput = (props: Partial<Props>) => {
   return React.createElement("input", inputProps);
 };
 
-export default React.memo(HexInput);
+export default React.memo<Props>(HexInput);
```

Comment:
I guess, we should make the `Props` type partial here, since `onChage` and `color` are optional properties (see https://github.com/omgovich/react-colorful/blob/e2444497b4573fd6c023fcb6e177107b6991426f/src/components/HexInput.tsx#L13):
```suggestion
export default React.memo<Partial<Props>>(HexInput);
```

Reply:
Sounds reasonable, thanks~

Reply:
Not just `onChange` and `color`, but every key in `HTMLInputElement` as `Props` extends it.